spec/02-integration/04-admin_api/09-routes_routes_spec.lua

```diff
@@ -282,11 +282,6 @@ for _, strategy in helpers.each_strategy() do
               headers = { ["Content-Type"] = content_type }
             })
 
-            -- TODO: For some reason the body which arrives to the server is
```

Comment:
@kikito This comment doesn't apply here (we are not passing the `methods` field), but I'm happy to look into this in other tests where the comment applies.